IMAGE:
  WIDTH: 640
  HEIGHT: 480
  ROI_START_HEIGHT: 320 # 320
  ROI_HEIGHT: 50  # 50

XYCAR:
  START_SPEED: 0.0
  MAX_SPEED: 30.0 # 15.0
  MIN_SPEED: 6.5 # 5.0
  SPEED_CONTROL_THRESHOLD: 10.0 # 15.0
  ACCELERATION_STEP: 0.05 # 0.05
  DECELERATION_STEP: 0.15  # 0.1

PID:
  P_GAIN: 0.22  # 0.20
  I_GAIN: 0.001  # 0.0006
  D_GAIN: 0.17  # 0.08

STANLEY:
  K_GAIN: 0.032 # 0.032
  LOOK_AHEAD_DISTANCE: 0.2 # 0.2

MOVING_AVERAGE_FILTER:
  SAMPLE_SIZE: 30 # 30

TOPIC:
  PUB_NAME: /xycar_motor
  SUB_NAME: /usb_cam/image_raw/
  QUEUE_SIZE: 1

CANNY:
  LOW_THRESHOLD: 200  # 200
  HIGH_THRESHOLD: 250 # 250

HOUGH:
  ABS_SLOPE_RANGE: 10.0  # 5.0
  THRESHOLD: 35 # 45
  MIN_LINE_LENGTH: 30 # 40
  MAX_LINE_GAP: 10  # 10
  LANE_WIDTH: 20    # 40

DEBUG: true
